build.sh

```diff
@@ -15,17 +15,19 @@ print_help_and_exit () {
 "./build.sh [options]
 
 List of options:
-  --release | -r                    : build in \"Release\" mode (default is \"Debug\")
-  --skip-polygeist <polygeist-path> : skip building POLYGEIST
-  --visual-dataflow | -v            : build visual-dataflow's C++ library
-  --export-godot | -e <godot-path>  : export the Godot project (requires engine)
-  --force | -f                      : force cmake reconfiguration in each (sub)project
-  --threads | -t <num-threads>      : number of concurrent threads to build on (by
-                                      default, one thread per logical core on the host
-                                      machine)
-  --disable-build-opt | -o          : don't use clang/lld/ccache to speed up builds
-  --check | -c                      : run tests during build
-  --help | -h                       : display this help message
+  --release | -r                       : build in \"Release\" mode (default is \"Debug\")
+  --skip-polygeist <polygeist-path>    : skip building POLYGEIST
+  --visual-dataflow | -v               : build visual-dataflow's C++ library
+  --export-godot | -e <godot-path>     : export the Godot project (requires engine)
+  --force | -f                         : force cmake reconfiguration in each (sub)project
+  --threads | -t <num-threads>         : number of concurrent threads to build on (by
+                                         default, one thread per logical core on the host
+                                         machine)
+  --llvm-parallel-link-jobs <num-jobs> : Maximum number of simultaneous link jobs started
```

Comment:
nit pick: Start description with a lower case letter for consistency. I also think "started" doesn't add meaning here (and might confuse some people), "simultaneous" already means what you want.

Reply:
Good catch. Updated.